### DISCUSSION TITLE
Failed to start Network Time Synchronization

### DISCUSSION THREAD

**Discussion by r-kurvits** [ltsp/ltsp#258] [Q&A]
Hello!

So i set up an 2 nic Ubuntu 18.04 ltsp server. I used server image and installed ubuntu-desktop on it. But the problem is with clients. When i boot up a thin client it will get stuck. It says "A start job is running for Network Time Synchronization (1s / 1min 30s). After time runs out it says "Failed to start Network Time Synchronization" and then time starts counting again.

**Comment by alkisg** [comment]
Hi nectal, which LTSP version?

```shell
dpkg -l | grep ltsp
```

Btw, you left from IRC too soon, give it some more time next time! :)

**Comment by r-kurvits** [comment]
Hi alkisg
`root@ltsp6:~# dpkg -l | grep ltsp
ii  ltsp                                       20.01-1~202001301202~ubuntu18.04.1               all          Linux Terminal Server Project
ii  ltsp-binaries                              19.11-1~201911110713~ubuntu18.04.1               all          Linux Terminal Server Project binaries`

**Comment by alkisg** [comment]
It sounds like something breaks the client IP. Are you using chrootless for the client image? If so, how did you configure networking on the server, with netplan?
Can you see the clients e.g. via epoptes at that point?

**Comment by r-kurvits** [comment]
I tried to create image with chroot but right now i don't know if i have chrootless image or image with chroot since i don't know how to access it. In the old ltsp it was easy. 
Command was 
> chroot /opt/ltsp/amd64/


And yes i configured network with netplan.
Also i don't use epoptes since i configure server over ssh and i don't know how do set up epoptes in my own computer so i can see clients.

**Comment by alkisg** [comment]
The command for chroot now is:

```shell
chroot /srv/ltsp/x86_64
```

...or whatever else your chroot name is. In general, it's easier to manage chrootless or even VM images than chroots, as it doesn't require knowing a lot of shell commands and tricks. If you can use chrootless, use it, it makes more sense than having to manage a desktop image via SSH/chroot.

What is the output of the following commands?

```shell
ltsp info
ls -lhd /srv/ltsp/*/proc /srv/ltsp/images/*
grep key.[0-9] /srv/tftp/ltsp/ltsp.ipxe
```

**Comment by r-kurvits** [comment]
```shell
root@ltsp6:~# ltsp info
LTSP version: 20.01-1~202001301202~ubuntu18.04.1
SERVER XSESSIONS: gnome-flashback-compiz.desktop
gnome-flashback-metacity.desktop
ubuntu-communitheme-snap.desktop
ubuntu.desktop
SERVER OS: PRETTY_NAME="Ubuntu 18.04.4 LTS"
CHROOTS:
VMs:
IMAGES:
x86_64

root@ltsp6:~# ls -lhd /srv/ltsp/*/proc /srv/ltsp/images/*
ls: cannot access '/srv/ltsp/*/proc': No such file or directory
-rw-r--r-- 1 root root 1,5G märts 10 15:03  /srv/ltsp/images/x86_64.img
-rw-r--r-- 1 root root 1,5G märts 10 14:23  /srv/ltsp/images/x86_64.img.old`

root@ltsp6:~# grep key.[0-9] /srv/tftp/ltsp/ltsp.ipxe
item --key 1 x86_64               x86_64.img`
```

**Comment by alkisg** [comment]
You have no chroot. You are using chrootless, `ltsp image /`.
Try putting this command in ltsp.conf:

```
[clients]
POST_INIT_RM_NETPLAN="rm -rf /etc/netplan"
```

Then run `ltsp initrd`, then reboot client. See if removing the netplan configuration that way fixes things.

**Comment by r-kurvits** [comment]
Tried that but no change.

**Comment by alkisg** [comment]
Did you do the NAT steps? Can your clients actually access the NTP server behind the server second NIC?
You may also come to IRC for live support, as I think this case will need 10x more time over github issues than IRC.

**Comment by r-kurvits** [comment]
If somebody has the same error you probably used same installation method as me. Like the old ltps. You install server and then all packages and then desktop image. This is the old way.
Tried with desktop image as documentation says and everything worked as it should.
Thank you for your time and help

Case closed.

**Comment by alkisg** [comment]
Installing with the desktop.iso has been the recommended method for LTSP5 as well: https://github.com/ltsp/ltsp5/wiki/Ubuntu-Install
